auction/auction.py

```diff
@@ -230,6 +230,49 @@ def __init__(self, bot):
 
             await self._bid(ctx, delta, user)
 
+    @auction.command(pass_context=True, no_pm=True)
+    async def bidders(self, ctx, user : discord.Member = None):
+        """Shows the current top 5 bidders"""
+        
+        server = ctx.message.server
+
+        if server.id not in self.data:
+            self.data[server.id] = {}
+            dataIO.save_json(self.file_path, self.data)
+
+        if user is None:
+            user = author
+
+        bidders = server[user.id]
```

Comment:
It should be `self.data[server.id][user.id]`. Also, you should check if `user.id in self.data[server.id]`. Use `server_data` to save typing :P